### PR TITLE
fixed #953 - updated the correct URL in documentation and website to obtain the 0.4 version

### LINKED ISSUE #953: Documentation mistake
version number is wrong in the download link which cause 404

this 

`$ wget https://github.com/meanjs/mean/archive/0.4.0.zip -O meanjs.zip; unzip meanjs.zip; rm meanjs.zip`

should be 

`$ wget https://github.com/meanjs/mean/archive/v0.4.0.zip -O meanjs.zip; unzip meanjs.zip; rm meanjs.zip`

I will submit pull request for it.

### PR DESCRIPTION
**Pull request by lirantal**
Closes https://github.com/meanjs/mean/issues/953

### CODE REVIEW

**Comment by lirantal** [comment]
@codydaig yours to review (it's a documentation update)

**Comment by codydaig** [comment]
@lirantal I think it should be v0.4.1 to match the latest.

**Comment by mleanos** [comment]
This says "..from the master branch.."
https://github.com/meanjs/mean/pull/981/files#diff-52d123e784c18e19488b96687c512943R60

Probably should match this with the version we're suggesting to download. I don't have a preference, as to which version we specify here. Master, v0.4.0, or v0.4.1 (latest) all seem appropriate; depending on our intention here.

**Comment by lirantal** [comment]
You're both right, it should be pointing to the latest indeed. I'll update.

Regards,
Liran Tal.

**Comment by lirantal** [comment]
Updated to reflect version v0.4.1

File: _includes/docs/0.4.x/gettingstarted.html [hunk 1]
```diff
@@ -59,7 +59,7 @@
   <p>
     Another way to use the MEAN.JS boilerplate is to download a zip copy from the master branch on github. You can also do this using wget command:
```
Comment (mleanos) [inline _includes/docs/0.4.x/gettingstarted.html]: @lirantal Not trying to be nit picky :) but this sentence still says "master branch".

Comment (lirantal) [inline _includes/docs/0.4.x/gettingstarted.html]: @mleanos  You should be as nit picky as required.
There's another line to update that I just noticed - I'll commit and push the change now.

**Comment by mleanos** [comment]
Great :) LGTM

Thank you, Liran!

**Comment by codydaig** [comment]
LGTM :+1:
Thanks!

**Pull request merged**